components/Hero.tsx

```diff
@@ -0,0 +1,43 @@
+import styles from "@/styles/Hero.module.css";
+import { textFont, titleFont } from "@/pages/_app";
+
+const Hero = () => {
+  return (
+    <div className={styles["hero"]}>
+      <div className={styles["hero__section"]}>
+        <div>
+          <div
+            data-testid="hero__slogan"
```

Comment:
It's better to use other queries than test-id, such as getByLabelText/getByPlaceholderText. Because test-id has the lowest priority. https://testing-library.com/docs/queries/about/

Reply:
Agree with Kerry, please check out the suggestions for getting the testing element in the testing playground (recommended in the readme file). 

Reply:
Good advice, thanks

Reply:
> It's better to use other queries than test-id, such as getByLabelText/getByPlaceholderText. Because test-id has the lowest priority. https://testing-library.com/docs/queries/about/

I have updated the testing queries. Please let me know if it needs to be further improved.